lib/events.js

```diff
@@ -413,16 +413,9 @@ function spliceOne(list, index) {
   list.pop();
 }
 
-function arrayClone(arr, len) {
-  var ret;
-  if (len === undefined)
-    len = arr.length;
-  if (len >= 50)
-    ret = arr.slice();
-  else {
-    ret = new Array(len);
-    for (var i = 0; i < len; i += 1)
-      ret[i] = arr[i];
-  }
-  return ret;
+function arrayClone(arr, i) {
+  var copy = new Array(i);
+  while (i--)
```

Comment:
Yeah, it's really micro, but I also always do this when possible/sane, idk I guess I just like the way it looks. XD